doc-site/components/nve-carousel.md

```diff
@@ -0,0 +1,214 @@
+  </nve-carousel-item>
+  <nve-carousel-item>
+    <img src="/assets/dark-mode-start-image.avif" alt="Bilde av naturen på natten." />
+  </nve-carousel-item>
+  <nve-carousel-item>
+    <img src="/assets/mønstre.png" alt="Mønstre" />
+  </nve-carousel-item>
+  <nve-carousel-item>
+    <img src="/assets/farger1.png" alt="Farger" />
+  </nve-carousel-item>
+</nve-carousel>
+```
+
+</CodeExamplePreview>
+
+### Bildetekst
+
+Bruk `description` på `nve-csarousel-item`for å legge til en bildetekst til hvert bilde om ønskelig. Det er mulig å bare ha bildetekst på noen av bildene.
+
+<CodeExamplePreview>
+
+```html
+<nve-carousel navigation pagination loop>
+  <nve-carousel-item description="Foto: Ola Nordmann.">
+    <img src="/assets/start-page-image.jpg" alt="Bilde av naturen i dagslys." />
+  </nve-carousel-item>
+  <nve-carousel-item description="Fjellrekken på natten.">
+    <img src="/assets/dark-mode-start-image.avif" alt="Bilde av naturen på natten." />
+  </nve-carousel-item>
+  <nve-carousel-item description="Bildekollage av mønstre.">
+    <img src="/assets/mønstre.png" alt="Mønstre" />
+  </nve-carousel-item>
+  <nve-carousel-item description="Sammensetting av farger.">
+    <img src="/assets/farger1.png" alt="Farger" />
+  </nve-carousel-item>
+</nve-carousel>
+```
+
+</CodeExamplePreview>
+
+### Thumbnails
+
+Bruk `nve-carousel-thumbnail` komponentet for å vise thumbnails under bildekarusellen. For å koble thumbnails opp mot riktig karusell, må det å defineres en `id` på `nve-carousel`, og `carouselId` på `nve-carousel-thumbnail`. Desse må være like.
+
+<CodeExamplePreview>
+
+```html
+<nve-carousel navigation loop id="thumbnail">
+  <nve-carousel-item>
+    <img src="/assets/start-page-image.jpg" alt="Bilde av naturen i dagslys." />
+  </nve-carousel-item>
+  <nve-carousel-item>
+    <img src="/assets/dark-mode-start-image.avif" alt="Bilde av naturen på natten." />
+  </nve-carousel-item>
+  <nve-carousel-item>
+    <img src="/assets/mønstre.png" alt="Mønstre" />
+  </nve-carousel-item>
+</nve-carousel>
+
+<nve-carousel-thumbnail carouselId="thumbnail">
+  <img src="/assets/start-page-image.jpg" alt="Bilde av naturen i dagslys." />
+  <img src="/assets/dark-mode-start-image.avif" alt="Bilde av naturen på natten." />
+  <img src="/assets/mønstre.png" alt="Mønstre" />
+</nve-carousel-thumbnail>
+```
+
+</CodeExamplePreview>
+
+### Autoplay
+
+Bruk `autoplay` til å rotere mellom bildene i karusellen automatisk.
+
+<CodeExamplePreview>
+
+```html
+<nve-carousel pagination autoplay loop>
+  <nve-carousel-item>
+    <img src="/assets/start-page-image.jpg" alt="Bilde av naturen i dagslys." />
+  </nve-carousel-item>
+  <nve-carousel-item>
+    <img src="/assets/dark-mode-start-image.avif" alt="Bilde av naturen på natten." />
+  </nve-carousel-item>
+  <nve-carousel-item>
+    <img src="/assets/mønstre.png" alt="Mønstre" />
+  </nve-carousel-item>
+</nve-carousel>
+```
+
+</CodeExamplePreview>
+
+### Flere slides per view
```

Comment:
Her innfører du begrepene _slides_ og  _view_.
Da begynner jeg å lure på hva et view er, hva en slide er og hva en side er. Trenger vi å bruke begrepene _view_ og _slide_ her?
PS. Om du trenger et begrep for _slide_, kan lysbilde være et alternativ.